.opencode/commands/career-ops-interview-prep.md

```
@@ -0,0 +1,12 @@
+---
+description: Full interview prep package for a specific company and role
+---
+
+Run full interview prep for the following company and role:
+
+$ARGUMENTS
+
+Load the career-ops skill:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)